endpoints.php

```diff
@@ -679,7 +679,9 @@ function addTapestryLink($request)
         if (!TapestryHelpers::userIsAllowed('ADD', $link->source, $postId)) {
             throw new TapestryError('ADD_NODE_PERMISSION_DENIED');
         }
-        if (!TapestryHelpers::userIsAllowed('ADD', $link->target, $postId)) {
+        $isNewLink = $link->value == 1;
```

Comment:
This line is a bit confusing and looks hacky. Why does this test on `link->value` show that it's a new link? And what exactly does it mean for a node to be new in this case?

Reply:
Originally, I was basing it off newLink in the NodeModal/index.vue to differentiate it from creating a link via add button:
```
          const newLink = {
            source: this.parent.id,
            target: id,
            value: 1,
            type: "",
          }
```
I added a property `addedOnNodeCreation` to clear that up. Let me know if that's still confusing.